src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/References/DesignTimeAssemblyResolutionTests.cs

```diff
@@ -195,7 +195,6 @@ public void ResolveAssemblyPathInTargetFx_NameThatMatches_ReturnsResolvedPaths(s
         [Theory]    // Input                                                                        // Name             // Version          // Path
         [InlineData("System",                                                                       "System.Core",      "",                 @"C:\System.Core.dll")]
         [InlineData("System",                                                                       "system",           "",                 @"C:\System.dll")]
-        [InlineData("system",                                                                       "System",           "",                 @"C:\System.dll")]
         [InlineData("encyclopædia",                                                                 "encyclopaedia",    "",                 @"C:\System.dll")]
```

Comment:
This is a bug that I need to figure out and wanted to avoid , basically AssemblyName in some situations is returning "System" when it parses "system" - this only happens on command-line. 

I assumed it was some sort of App Compat switch and we changed the way that AssemblyName was parsed between framework versions - but I ruled that out. I started debugging it via the command-line and it doesn't repro. I think it's a race condition in the CLR's parsing code - but I haven't got that far. There's not a long I can do about in my implementation - other than writing my own AssemblyName parser.